src/observable/dom/AjaxObservable.ts

```diff
@@ -417,7 +417,7 @@ export class AjaxResponse {
           this.response = JSON.parse(xhr.responseText || 'null');
         }
         break;
-      case 'xml':
+      case 'document':
```

Comment:
Because it seems that `xml` is not a valid responseType, both per-spec as well as per the TS 2.3 XHR typings (see the links I gave in the PR description). I don't think this code had test cases, unfortunately.